article/future-work.tex

```diff
@@ -1,9 +1,35 @@
 \section{Future work}
-For our master thesis we will continue to explore recommendation systems and continue in the path of LightGCN and Price-aware recommendation.
-There are several topics related to these implementation to be investigated to gain a better understanding of how recommender systems can be improved.
+For our master thesis, we will continue to explore recommendation systems and continue in the path of LightGCN and Price-aware recommendation.
+There are several topics related to these implementations to be investigated to gain a better understanding of how recommender systems can be improved.
 The hypothesis for the master thesis are:
 \begin{itemize}
-    \item Adding price and category embeddings to LightGCN will improve performance of the simple price aware extension.
+    \item Adding price and category embeddings to LightGCN will improve the performance of the simple price-aware extension.
     \item If adding price and category embeddings improves the performances, then it can get generalized to include other features than price and category.
     \item Price-aware recommendation can be generalized so other features can get integrated to improve the recommendation performance.
+    \item Extending categories so one item can have multiple categories will better capture cross-category price preferences.
 \end{itemize}
+
+\subsection{Adding price-aware embeddings to LightGCN}
+An extension of the embeddings in LightGCN could be implemented by combining the embeddings from the different methods.
+Inspired by utilizing \autoref{eq:price-aware-prediction}, we could use the embeddings for price and category.
+An alternative could be trying to combine the original LightGCN embedding without price and category for users and items, as these capture the collaborative filtering quite well and then adding the price and category embeddings.
+A possible outcome of this can however be that there still are contradictions between the collaborative signals, and price and category signals.
+LightGCN does not utilize feature transformation matrices and non-linear activation functions, which can make it not suited for handling side information.
+PUP and NGCF does use feature transformation matrices and non-linear activation functions, so if extending LightGCN is not feasible, we might be able to do it with PUP or NGCF.
+
+\subsection{Generalizing feature inputs for LightGCN or Price-Aware recommendation}
+The intuition behind Price-Aware recommendations can easily be put to use for other datasets with other types of data.
+Doing it for the adapted LightGCN requires that adding price and category embeddings gives a reasonable result before it makes sense to generalize it.
```

Comment:
```suggestion
Before it would make sense to generalize the feature inputs in the adapted LightGCN, we would first have to make sure that adding the price and category was actually beneficial for the method.
```